src/app.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import accountRoter from './routes/account-routes';
+
+const app = express();
+
+const dbName = 'FoodDelivery';
```

Comment:
Instead of hardcoding the connection details it's recommended to keep them in .env files. See https://github.com/motdotla/dotenv